modules/api/src/main/java/com/intuit/wasabi/api/ExperimentsResource.java

```diff
@@ -52,9 +66,7 @@
 import static java.lang.Boolean.FALSE;
 import static java.util.TimeZone.getTimeZone;
 import static javax.ws.rs.core.HttpHeaders.AUTHORIZATION;
-import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
-import static javax.ws.rs.core.MediaType.APPLICATION_JSON_TYPE;
-import static javax.ws.rs.core.MediaType.TEXT_PLAIN;
+import static javax.ws.rs.core.MediaType.*;
```

Comment:
This may be a strange IntelliJ-setting, but above you removed the wildcard for explicit imports and here it is added. Or do you want explicit for internal sources and implicit for external?